lib/features/selection/Selection.js

```diff
@@ -76,6 +77,15 @@ Selection.prototype.select = function(elements, add) {
     elements = elements ? [ elements ] : [];
   }
 
+  var canvas = this._canvas;
+
+  elements = elements.filter(function(element) {
+    var plane = canvas.findPlane(element);
+
+    // When the selected Element is not part of any plane, select it anyway
```

Comment:
Can you explain the reasoning behind this? If this behavior makes sense we should also have a test for it.

Reply:
When moving an element, it is not part of any plane, but should still be selected after being dropped. It is tested in this test case: https://github.com/bpmn-io/diagram-js/blob/develop/test/spec/features/selection/SelectionBehaviorSpec.js#L278

Reply:
Why is not part of any plane? Is it not a child of a root element that is connected to a plane?

Reply:
I had another closer look and I misunderstood what was happening. The dragged element will be part of the Plane, the issue was with the test which created an impossible scenario. It lead to the moved shape being its own parent and therefore not being part of any plane.

I adjusted the Test to be applicable and removed the workarounds in the code: https://github.com/bpmn-io/diagram-js/pull/561/commits/ceaeac1f8a14a11cbf4563976d262cc48db3e2b1